test/dedup-id/package.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Not sure on the acceptance test location. What differentiates it from the rest of the tests in this folder. I believe that some are integration tests (i.e. meant to be run only locally), or am I mistaken.